packages/components/src/menu-item/index.js

```diff
@@ -74,15 +74,21 @@ export function MenuItem( {
 		props.icon = icon;
 	}
 
+	const atts = {
```

Comment:
Convention is that this should be called 'props' instead of `atts`. I think `childProps` is a good name given `props` is already used in this scope. Alternatively could avoid extracting the props into a variable and keep them inline:

```javascript
const supportsAriaChecked = role === 'menuitemcheckbox' || role === 'menuitemradio';
const ariaChecked = supportsAriaChecked  ? isSelected : undefined;
// ... snip
return createElement(
		tagName,
		{
			'aria-label': label,	
			'aria-checked': ariaChecked,	
			role,	
			className,	
			...props,	
		},
```